{{ cookiecutter.formal_name }}/briefcase.toml

```diff
@@ -1,14 +1,20 @@
 # Generated using Python {{ cookiecutter.python_version }}
 [paths]
-app_path = "{{ cookiecutter.formal_name }}/app"
-app_packages_path = "{{ cookiecutter.formal_name }}/app_packages"
+app_path = "{{ cookiecutter.class_name }}/app"
+app_packages_path = "{{ cookiecutter.class_name }}/app_packages"
+info_plist_path = "{{ cookiecutter.class_name }}/Info.plist"
```

Comment:
The plist path definitely isn't required for this change - although it's probably helpful to keep it, since we're going to need it when we get to permission handling.

The app template PR would need a binary update, but we could do that with a regular tagging; however, again, we'd need to reserve the plist for permissions purposes.